packages/manager/apps/pci-rancher/package.json

```diff
@@ -12,9 +12,9 @@
   "author": "OVH SAS",
   "scripts": {
     "beta-test:e2e": "tsc && npx playwright test --headed",
-    "build": "tsc && vite build",
+    "build": "vite build",
```

Comment:
Without tsc you will loose typechecking, I think you need to use the same commands as other projects instead :
`tsc --project tsconfig.build.json && vite build`